pkg/server/mrt.go

```diff
@@ -282,15 +282,15 @@ func (m *mrtWriter) loop() error {
 	}
 }
 
-func mrtFileOpen(filename string, interval uint64) (*os.File, error) {
+func mrtFileOpen(filename string, dInterval uint64) (*os.File, error) {
 	realname := filename
```

Comment:
Two callers of mrtFileOpen() uses rotationInterval for the interval argument?
https://github.com/osrg/gobgp/blob/master/pkg/server/mrt.go#L253
https://github.com/osrg/gobgp/blob/master/pkg/server/mrt.go#L328

Reply:
Thanks for your review. revised and pushed again.